tests/src/test/java/alluxio/client/FileOutStreamIntegrationTest.java

```diff
@@ -139,6 +146,7 @@ public void outOfOrderWriteTest() throws Exception {
     os.write(BufferUtils.getIncreasingByteArray(1, length));
     os.close();
 
-    checkWrite(filePath, mWriteAlluxio.getUnderStorageType(), length + 1, length + 1);
+    checkWrite(filePath, CreateFileOptions.defaults().setWriteType(WriteType.MUST_CACHE).getUnderStorageType(),
```

Comment:
`CreateFileOptions.defaults().setWriteType(WriteType.MUST_CACHE).getUnderStorageType()` is equivalent to `UnderStorageType.NO_PERSIST`